client/src/components/IconTextField.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Box, Grid, InputAdornment, TextField } from "@mui/material";
+
+const IconTextField = ({ icon, placeholder, value, setValue }) => {
+  return (
+    <Grid item sx={{ backgroundColor: "white", borderRadius: "20rem" }}>
+      <TextField
+        hiddenLabel
+        fullWidth
+        placeholder={placeholder}
+        value={value}
+        onChange={(e) => setValue(e.target.value)}
+        InputProps={{
+          startAdornment: (
+            <Box padding="0.3rem">
+              <InputAdornment position="start">{icon}</InputAdornment>
+            </Box>
+          ),
+        }}
+        sx={{
+          "& .MuiOutlinedInput-root": {
+            "& fieldset": {
+              border: "none",
+              borderRadius: "20rem",
+            },
+          },
+          "& .MuiInputBase-input": {
+            fontSize: ".95rem",
+          },
```

Comment:
Are these styles we'll want to override for every input so it matches the designs? .95 rem also seems very specific

Reply:
I don't think so...I believe there are other styles for text input as well. I can only think of one other occurrence where it would be re-used (which is another search bar but without the icons -  but I think for this we can maybe reuse this component)